config/spryk/templates/Common/Method.php.twig

```diff
@@ -23,7 +23,8 @@
      {% endif -%}
      * @return {{ output }}
      */
-    public function {{ method }}({% if input.value is not empty %}{{ input.value | typedArrayConvert }}{% endif %}): {{ output | typedArrayConvert }}
+    public function {{ method }}({% if input.value is not empty %}{{ input.value | typedArrayConvert }}{% endif %}){% if output.value is not empty %}: {{ output | typedArrayConvert }}{% endif %}
```

Comment:
I can't change this
`{% if output.value is not empty %}: {{ output | typedArrayConvert }}{% endif %}`
to this
`{{ output.value | default('') }}`
because we need to add also ": " if return type not empty